src/app/index.css

```diff
@@ -59,3 +59,20 @@ body {
   src: url("../shared/ui/assets/fonts/Mulish-ExtraBold.ttf");
   font-weight: 800;
 }
+
```

Comment:
We have tailwind in our project, no need to use css. You can create css file inside directory that requires it, but in this case I don't see reason to use css(especially in index.css).